client/blocks/login/index.jsx

```diff
@@ -57,22 +63,28 @@ class Login extends Component {
 	};
 
 	handleValidUsernamePassword = () => {
-		if ( ! this.props.twoFactorEnabled ) {
-			this.rebootAfterLogin();
-		} else {
+		if ( this.props.twoFactorEnabled ) {
```

Comment:
I'm maybe missing something here, but once the user gone to twoFactor screens, how he's getting back to the linking social screen?